client/container_diff_test.go

```diff
@@ -31,13 +31,13 @@ func TestContainerDiff(t *testing.T) {
 			if !strings.HasPrefix(req.URL.Path, expectedURL) {
 				return nil, fmt.Errorf("Expected URL '%s', got '%s'", expectedURL, req.URL)
 			}
-			b, err := json.Marshal([]container.ContainerChangeResponseItem{
+			b, err := json.Marshal([]container.FilesystemChange{
 				{
-					Kind: 0,
+					Kind: container.ChangeModify,
 					Path: "/path/1",
 				},
 				{
-					Kind: 1,
+					Kind: container.ChangeModify,
```

Comment:
Did you change this one on purpose? :eyes:

```suggestion
					Kind: container.ChangeAdd,
```

Reply:
oh! no, I don't think I did so on purposed. Guess that also shows how great of a test this is 🤔.

Reply:
"did we get back two items?  yes?  must be right"